From d71ad4f86fe0c356920cf80281b79ee4906ed508 Mon Sep 17 00:00:00 2001
From: Natanael Copa <ncopa@alpinelinux.org>
Date: Thu, 22 Dec 2016 13:25:49 +0000
Subject: [PATCH] Fix build with musl libc

Use standard integer type instead of uint. This fixes the following
error which occurs with musl libc:

bayer.c: In function 'dc1394_bayer_Downsample':
bayer.c:778:2: error: unknown type name 'uint'
  uint tmp;
  ^~~~

Signed-off-by: Natanael Copa <ncopa@alpinelinux.org>
---
 libdc1394/dc1394/bayer.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libdc1394/dc1394/bayer.c b/libdc1394/dc1394/bayer.c
index 10a4c1b..c348c24 100644
--- a/dc1394/bayer.c
+++ b/dc1394/bayer.c
@@ -775,7 +775,7 @@ dc1394_bayer_Downsample(const uint8_t *restrict bayer, uint8_t *restrict rgb, in
 {
 	uint8_t *outR, *outG, *outB;
 	register int i, j;
-	uint tmp;
+	unsigned int tmp;
 	int st=sx*sy;
 	int p;
 	int sx2=sx<<1;
-- 
2.11.0

